<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpzioniDialog</class>
 <widget class="QDialog" name="OpzioniDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Opzioni</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="viewMode">
          <enum>QListView::ListMode</enum>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>Generali</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/resource/jacksms_icona_impostazioni_24x24-7.png</normaloff>:/resource/jacksms_icona_impostazioni_24x24-7.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Rete</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/resource/jacksms_icona_statistiche_24x24-2.png</normaloff>:/resource/jacksms_icona_statistiche_24x24-2.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>JackSMS Messenger</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/resource/jacksms_icona_im_24x24-3.png</normaloff>:/resource/jacksms_icona_im_24x24-3.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Captcha</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/resource/plugin.png</normaloff>:/resource/plugin.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Lingua</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/resource/6.png</normaloff>:/resource/6.png</iconset>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="generali">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QCheckBox" name="checkSalvalocale">
               <property name="text">
                <string>Salva i messaggi sms in locale</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkAccountDefault">
               <property name="text">
                <string>Imposta l'account di default quando selezioni un contatto</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="errorSmsPopup">
               <property name="text">
                <string>Segnala con un popup l'errore sull'invio di un SMS oJMS</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="successSmsPopup">
               <property name="text">
                <string>Segnala con un popup il corretto invio di un SMS o JMS</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="captchaPopup">
               <property name="text">
                <string>Segnala con un popup la richiesta di inserimento captcha</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="nonSalvaCookies">
               <property name="toolTip">
                <string>Questa opzione disattiva il salvataggio delle sessioni. Utile per coloro che hanno spesso problemi di sessioni corrotte.</string>
               </property>
               <property name="text">
                <string>Disattiva il salvataggio delle sessioni (cookies)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="opzAutoLogin">
               <property name="toolTip">
                <string>Questa opzione viene disattivata automaticamente se si usa il tasto logout dal menu.</string>
               </property>
               <property name="text">
                <string>Effettua il login automaticamente</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="opzSvuotaInvioCorretto">
               <property name="text">
                <string>Pulisci la schermata per un nuovo messaggio dopo invio corretto </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="hideServiceUpdate">
               <property name="text">
                <string>Nascondi i messaggi di aggiornamento servizi</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="rete">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QCheckBox" name="CheckUsaProxy">
               <property name="text">
                <string>Utilizza un server proxy</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="1">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Porta</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLineEdit" name="TextServer"/>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="TextPort"/>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Server</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Tipo server</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QRadioButton" name="RadioHttp">
                 <property name="text">
                  <string>Http</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="RadioSocks5">
                 <property name="text">
                  <string>Socks v5</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QCheckBox" name="usaAutenticazione">
               <property name="text">
                <string>Abilita l'autenticazione proxy</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Username</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Password</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLineEdit" name="proxyUsername"/>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="proxyPassword">
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="im">
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QCheckBox" name="CheckAbilitaIM">
               <property name="text">
                <string>Attiva ricezione messaggi JMS all'avvio di JackSMS</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="suonoJMS">
               <property name="text">
                <string>Emetti un suono alla ricezione di un nuovo JMS</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="captchaOptionsPage">
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <widget class="QCheckBox" name="checkUseCaptcha">
               <property name="text">
                <string>Attiva riconoscimento CAPTCHA se è disponibile un plugin
appropriato</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>Zoom dell'immagine CAPTCHA</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboZoomCaptcha">
                 <item>
                  <property name="text">
                   <string>100%</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>120%</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>140%</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>160%</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>180%</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>200%</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>250%</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>300%</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>350%</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_6">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Lingua di sistema</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="languageComboBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Ok</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/resource/salva.png</normaloff>:/resource/salva.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Applica</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/resource/salva.png</normaloff>:/resource/salva.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Annulla</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/resource/rimuovi.png</normaloff>:/resource/rimuovi.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
